README.md

```diff
@@ -1,26 +1,102 @@
+
+#### Vector Tiles
+Vector tiles of your data can be generated at the endpoint `/vector/{z}/{x}/{y}`. Filtering by layer works via query string but can also be handled client side, as can styling. [See here for more info about the Mapbox Vector Tile specification.](https://www.mapbox.com/vector-tiles/)
+
+#### Debugging
+The local development server exposes a Node.js process WebSocket that can be attached to your IDE of choice to step through and debug your code. Here are instructions on how to do so using Google Chrome's Dev Tools:
+ * Start the development server with `./scripts/server`.
+ * Open Google Chrome and navigate to `about:inspect`.
+ * There should be an option listed as something along the lines of "Target (v8.10.0)" with the Node.js logo and a path like `file:///home/tiler/bin`. Click "inspect", underneath.
+ * A new window will open up. Enter `ctrl+p` to open a search bar that lets you navigate to your source code. _The transpiled code in `src/tiler/bin/` is what is actually being tracked by the debugger,_  not the source code in `src/tiler/src/`.
+
+### Deployment to AWS
+ * Specify your production credentials and lambda function settings in `.env`. The following variables are required:
+   * `AWS_PROFILE`: the name of the AWS user profile you want to deploy your project as. You may have this set already in your environment, otherwise you’ll want to set it to one of the names of the sets of credentials specified in `~/.aws/credentials`. Defaults to “default”.
+   * `PROJECT_NAME`: the name of your project. This must be unique among the functions you currently have deployed to AWS Lambda.
+   * `PROD_TILEGARDEN_*`: the credentials necessary to connect to your production database.
+   * `LAMBDA_REGION`: the AWS region to which you want your lambda functions deployed.
+   * `LAMBDA_ROLE`: the name/ARN of the IAM role to give your lambda. Leave this blank to have one created for you automatically.
+   * `LAMBDA_SUBNETS` and `LAMBDA_SECURITY_GROUPS`: only required if you need to connect to other AWS resources (such as an RDS instance), in which case these should match the values that those resources have.
+   * [See below for more options.](#Additional-Configuration-Options)
+ * [See below to make sure your AWS profile has the requisite permissions for automated deployment.](#Required-AWS-Permissions)
+ * Run `./scripts/publish --new`. This will:
+   1. Create a new AWS Lambda function with your specified settings
+   2. Create an API Gateway from which to serve your content
+   3. Create a CloudFront distribution to act as a proxy to circumvent certain API Gateway limitations.
+ * Updates to an existing project can be deployed by running `./scripts/publish` (no “--new”).
```

Comment:
It seems worth noting here what parameters can be changed by republishing and what ones require that you `destroy` the `publish --new` (unless I'm behind the times and republishing works for all configurable settings now).